samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-session.ipynb

```diff
@@ -0,0 +1,416 @@
+        "editable": true,
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "run_control": {
+          "frozen": false
+        }
+      },
+      "source": [
+        "## Run on an Azure Quantum backend\n",
+        "Now, you will run the same problem on a backend of your choice through Azure Quantum. Here the maximum number of iterations of the optimizer is set to 50.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "from azure.quantum.qiskit import AzureQuantumProvider\n",
+        "\n",
+        "# Connect to the Azure Quantum workspace via a Qiskit provider\n",
+        "provider = AzureQuantumProvider(\n",
+        "            resource_id = \"\",\n",
+        "            location = \"\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "ionq_sim = provider.get_backend('ionq.simulator')\n",
+        "quantinuum_sim = provider.get_backend('quantinuum.sim.h1-1e')\n",
+        "rigetti_sim = provider.get_backend('rigetti.sim.qvm')\n",
+        "\n",
+        "# Set the backend you want to use here.\n",
+        "backend_to_use = ionq_sim"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "You will now kickoff the VQE run on the selected backend through Azure Quantum. Since this will trigger several jobs, a `session` is used to group them together into a single logical entity.\n",
+        "\n",
+        "**IMPORTANT !!**\n",
+        "\n",
+        "1. This cell will take about 20 minutes to run on the IonQ simulator for max_iter=50. It generates around 300 jobs. The time can vary depending on the backend queue times. Running with max_iter=50 may not be sufficient for the results to converge. You could consider increasing the number of iterations to 100 to give a more accurate result, but please be aware of the increased running times for the cell.\n",
+        "1. If you are an Azure Quantum credits user, you may not have enough credits to run this sample on certain backends.\n",
+        "1. If you run this against a QPU hardware backend instead of a simulator, you will likely incur a large cost or consume a large number of your alloted credits.\n",
```

Comment:
```suggestion
        "1. **If you run this against a QPU hardware backend instead of a simulator, you will likely incur a large cost or consume a large number of your alloted credits.**\n",
```